fe/fe-core/src/main/java/com/starrocks/catalog/Type.java

```diff
@@ -691,30 +691,43 @@ public static Type fromPrimitiveType(PrimitiveType type) {
 
     public boolean canApplyToNumeric() {
         // TODO(mofei) support sum, avg for JSON
-        return !isOnlyMetricType() && !isJsonType() && !isFunctionType() && !isStructType() && !isMapType();
+        return !isOnlyMetricType() && !isJsonType() && !isFunctionType() && !isStructType() &&
```

Comment:
Is it better to use confirmation type than to use non, otherwise, if you add a type, you have to change it.？

Reply:
the fact already been this...